presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionTreeUtils.java

```diff
@@ -169,4 +171,12 @@ private static boolean isEnumLiteral(DereferenceExpression node, Type nodeType)
         Object enumValue = enumType.getEnumMap().get(enumKey);
         return enumValue instanceof String ? Optional.of(utf8Slice((String) enumValue)) : Optional.of(enumValue);
     }
+
+    public static FieldId checkAndGetFieldReferenceForExpression(Expression expression, Multimap<NodeRef<Expression>, FieldId> columnReferences)
```

Comment:
It makes more sense to use "column reference" rather than "field reference". Or maybe `checkAndGetColumnReferenceField`.

Reply:
Done